README.md

```diff
@@ -1,14 +1,59 @@
 # Apple Support for [Bazel](https://bazel.build)
 
-This repository contains helper methods that support building rules that
-target Apple platforms. See [the
-docs](https://github.com/bazelbuild/apple_support/tree/master/doc) for
-how you can use these helpers. Also see
-[rules_apple](https://github.com/bazelbuild/rules_apple) and
-[rules_swift](https://github.com/bazelbuild/rules_swift) for more Apple
-platform rules.
-
-## Quick setup
-
-Copy the `WORKSPACE` snippet from [the releases
-page](https://github.com/bazelbuild/apple_support/releases).
+This repository contains the [Apple CC toolchain](#toolchain-setup),
+Apple related [platforms](platforms/BUILD) and
+[constraints](constraints/BUILD) definitions, and small helper functions
+for rules authors targeting Apple platforms.
+
+If you want to build iOS, tvOS, watchOS, or macOS apps, use
+[`rules_apple`][rules_apple].
+
+If you want to build Swift use
+[`rules_swift`](https://github.com/bazelbuild/rules_swift).
+
+See [the documentation](doc) for the helper rules provided by this
+repository.
+
+## Installation
+
+Copy the `MODULE.bazel` or `WORKSPACE` snippets from [the releases
+page](https://github.com/bazelbuild/apple_support/releases) into your
+project.
+
+## Toolchain setup
+
+The Apple CC toolchain in by this repository provides toolchains for
+building for Apple platforms besides macOS. Since Bazel 7 this toolchain
+is required when targeting those platforms but the toolchain also
+supports Bazel 6.
+
+To use the Apple CC toolchain, pull this repository into your build and
+add this to your `.bazelrc`:
+
+```bzl
+build --enable_platform_specific_config
+build:macos --apple_crosstool_top=@local_config_apple_cc//:toolchain
+build:macos --crosstool_top=@local_config_apple_cc//:toolchain
+build:macos --host_crosstool_top=@local_config_apple_cc//:toolchain
```

Comment:
```suggestion
build --apple_crosstool_top=@local_config_apple_cc//:toolchain
build --crosstool_top=@local_config_apple_cc//:toolchain
build --host_crosstool_top=@local_config_apple_cc//:toolchain
```

Reply:
If you build on Linux I don't think you should set these there though?

Reply:
Ahh I thought this is meant for the macOS platform. So this is like a base config for all Apple platforms? I think we can just mention the flags instead of an example .bazelrc.

Reply:
it is, but if your project builds on both platforms, setting this globally might cause issues